language/documentation/book/src/functions.md

```diff
@@ -43,7 +43,7 @@ script {
 }
 ```
 
-To allow access from other modules or from scripts, the function must be declared `public`, `public(friend)`, or `public(script)`.
+To allow access from other modules or from scripts, the function must be declared `public` or `public(friend)`.
```

Comment:
`public entry` also can be called from other modules right?

Reply:
`entry` is not a part of visibility, so would be more confusing to mention it than to not. 

You can have `public entry` and `public(friend) entry` and even (private/internal) `entry`